makefile

```diff
@@ -110,6 +112,22 @@ ios-arm-release: .build/projects/gmake-ios-arm
 	make -R -C .build/projects/gmake-ios-arm config=release
 ios-arm: ios-arm-debug ios-arm-release
 
+.build/projects/gmake-xros-arm64:
+	$(GENIE) --gcc=xros-arm64 gmake
+xros-arm64-debug: .build/projects/gmake-xros-arm64
+	make -R -C .build/projects/gmake-xros-arm64 config=debug
+xros-arm64-release: .build/projects/gmake-xros-arm64
+	make -R -C .build/projects/gmake-xros-arm64 config=release
+xros-arm64: xros-arm64-debug xros-arm64-release
+
+.build/projects/gmake-xros-simulator:
+	$(GENIE) --gcc=xros-simulator gmake
+xros-simulator-debug: .build/projects/gmake-simulator-simulator
+	make -R -C .build/projects/gmake-xros-simulator config=debug
+xros-simulator-release: .build/projects/gmake-simulator-simulator
+	make -R -C .build/projects/gmake-xros-simulator config=release
+xros-simulator: xros-simulator-debug xros-simulator-release
+
```

Comment:
No need to add this to makefile. Purpose of makefile in bgfx is just to give quick start, but not every platform has to be there. Not having it simplifies maintenance.